simulariumio/tests/converters/test_springsalad_converter.py

```diff
@@ -237,6 +238,297 @@
                 "plotData": {"version": CURRENT_VERSION.PLOT_DATA, "data": []},
             },
         ),
+        # truncated data from tutorial example (and draw bonds)
+        (
+            SpringsaladData(
+                sim_view_txt_file=InputFileData(
+                    file_path=(
+                        "simulariumio/tests/data/springsalad/"
+                        "Simulation0_SIM_VIEW_Run0.txt"
+                    ),
+                ),
+                meta_data=MetaData(
+                    scale_factor=0.1,
+                ),
+                display_data={
+                    "GREEN": DisplayData(
+                        name="A",
+                        radius=10.0,
+                        display_type=DISPLAY_TYPE.OBJ,
+                        url="a.obj",
+                        color="#dfdacd",
+                    ),
+                    "RED": DisplayData(
+                        name="B",
+                        color="#0080ff",
+                    ),
+                },
+            ),
+            {
```

Comment:
Is the test data from here (ln 267) down the same as what's in ln 45 to 238? If so maybe it'd be nice to save it into a variable to reuse?